File: lighthouse/src/main/java/com/automation/center/lighthouse/dto/testSuite/TestSuiteDto.java
package com.automation.center.lighthouse.dto.testSuite;

import com.automation.center.lighthouse.dto.resultHistory.ResultHistoryDto;
import lombok.Data;

import java.time.LocalDateTime;
import java.util.List;

@Data
public class TestSuiteDto {
    private Long id;
    private LocalDateTime startDate;
    private LocalDateTime endDate;
    private List<ResultHistoryDto> resultHistories;
}

File: lighthouse/src/main/java/com/automation/center/lighthouse/dto/testSuite/TestSuiteDto.kt
package com.automation.center.lighthouse.dto.testSuite

import com.automation.center.lighthouse.dto.metric.MetricDto
import com.automation.center.lighthouse.dto.suiteUrl.SuiteUrlDto

data class TestSuiteDto(
    var id: Long? = null,
    var name: String? = null,
    var description: String? = null,
    var cron: String? = null,
    val metrics: List<MetricDto> = ArrayList(),
    val urls: List<SuiteUrlDto> = ArrayList()
)
